### PR TITLE
message_referrer_report.py

### PR DESCRIPTION
**Pull request by annstriganova**

### CODE REVIEW

File: examples/message_referrer_report.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# Query the messages and save response to DataFrame with specified columns
+ef = EntityFilter(entity="*")
+df = DateFilter(start_date=startDate, end_date=endDate)
+query = MessageQuery(entity_filter=ef, date_filter=df, type="web", source="access.log", tags=tags, limit=10000)
+columns = ['date', 'request_uri', 'http_referer', 'remote_addr', 'geoip_org']
+message_frame = service.query_dataframe(query, columns=columns)
+
+# Exclude rows with specified tag values
+ignore_hosts = '|'.join(
```
Comment (raipc) [inline examples/message_referrer_report.py]: '|'.join(re.escape(s) for s in ('google', 'axibase.com', ...))

Comment (annstriganova) [reply examples/message_referrer_report.py]: @raipc, Thank you!

File: README.md [hunk 2]
```diff
@@ -442,9 +448,22 @@ Install the [`pandas`](http://pandas.pydata.org/) module for advanced data manip
 pip install pandas
 ```
 
+Use Pandas [`set_option`](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.set_option.html#pandas.set_option) to locate correct formation options:
```
Comment (rodionos) [inline README.md]: `to format output` is better.

**Comment by annstriganova** [comment]
No warnings are generated.

**Pull request merged**